src/main/hbase_region_inspector/hbase.clj

```diff
@@ -3,21 +3,22 @@
             [hbase-region-inspector.util :as util]
             [hbase-region-inspector.hbase.impl :as hbase-impl])
   (:import [org.apache.hadoop.hbase
-            client.HBaseAdmin util.Bytes
-            HBaseConfiguration ClusterStatus]
+            client.HBaseAdmin client.Admin util.Bytes
+            HBaseConfiguration ClusterStatus
+            ServerLoad]
            org.apache.hadoop.security.UserGroupInformation
            org.apache.hadoop.conf.Configuration))
 
 (defn server-load->map
   "Transforms ServerLoad object into clojure map"
   [load]
   (assoc
-    {:max-heap-mb        (.getMaxHeapMB load)
-     :used-heap-mb       (.getUsedHeapMB load)
-     :regions            (.getNumberOfRegions load)
-     :requests-rate      (.getNumberOfRequests load)
-     :store-files        (.getStorefiles load)
-     :store-file-size-mb (.getStorefileSizeInMB load)}
+    {:max-heap-mb        (.getMaxHeapMB ^ServerLoad load)
+     :used-heap-mb       (.getUsedHeapMB ^ServerLoad load)
+     :regions            (.getNumberOfRegions ^ServerLoad load)
+     :requests-rate      (.getNumberOfRequests ^ServerLoad load)
+     :store-files        (.getStorefiles ^ServerLoad load)
+     :store-file-size-mb (.getStorefileSizeInMB ^ServerLoad load)}
```

Comment:
It would be simpler to put the type hint on the parameter instead. i.e. `[^ServerLoad load]`

Reply:
thanks, done